recipes/stringzilla/all/test_package/test_package.cpp

```diff
@@ -1,31 +1,31 @@
 #include <cstdlib>
 #include <iostream>
+#if STRINGZILLA_API >= 3
+#include <array>
+#include <cstdint>
+#include <vector>
+#include "stringzilla/stringzilla.hpp"
+#else
 #include "stringzilla.h"
+#endif
 
-#ifdef STRINGZILLA_LESS_2_0
-
-int main(void) {
-  // Initialize your haystack and needle
-  strzl_haystack_t haystack = {
-    "Fastest string sort, search, split, "
-    "and shuffle for long strings and multi-gigabyte files in Python and C, "
-    "leveraging SIMD with Arm Neon and x86 AVX2 & AVX-512 intrinsics.",
-    171};
-  strzl_needle_t needle = {"SIMD", 4};
+#if STRINGZILLA_API >= 3
 
-  // Count occurrences of a character like a boss 😎
-  size_t count = strzl_naive_count_char(haystack, 'a');
+namespace sz = ashvardanian::stringzilla;
 
-  // Find a character like you're searching for treasure 🏴‍☠️
-  size_t position = strzl_naive_find_char(haystack, 'a');
+int main(void) {
+  sz::string haystack = "some string";
+  sz::string_view needle = sz::string_view(haystack).substr(0, 4);
 
-  // Find a substring like it's Waldo 🕵️‍♂️
-  size_t substring_position = strzl_naive_find_substr(haystack, needle);
+  auto substring_position = haystack.find(needle); // Or `rfind`
 
-  return EXIT_SUCCESS;
+  haystack.end() - haystack.begin() == haystack.size(); // Or `rbegin`, `rend`
+  haystack.find_first_of(" \w\t") == 4; // Or `find_last_of`, `find_first_not_of`, `find_last_not_of`
+  haystack.starts_with(needle) == true; // Or `ends_with`
+  haystack.remove_prefix(needle.size()); // Why is this operation in-place?!
```

Comment:
Minor, for consistency add `return EXIT_SUCCESS;`.

Reply:
fixed.